app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ActiveRecord::Base
+  belongs_to :user
```

Comment:
A like belongs to an `author` as you use a `custom foreign key` named `author_id` in likes table.
`For example: belongs_to :author, class_name: 'User'`